cloud_images/centos7/create_base_ami.sh

```diff
@@ -53,7 +53,9 @@ ONBOOT="yes"
 TYPE="Ethernet"
 USERCTL="yes"
 PEERDNS="yes"
-IPV6INIT="no"
+IPV6INIT="yes"
+DHCPV6C="yes"
+NM_CONTROLLED="no"
```

Comment:
It shouldn't have been used in the first place. NetworkManager on EL7 is v1.4.0 (vs. current stable v1.8.0). NM is handy on a laptop moving between multiple networks. Doesn't add any value on a server with static configuration and where you don't need users be able to change the networking. We even turn off firewalld so no gain there either. One less moving part, one less source of errors.